packages/dnb-eufemia/src/extensions/payment-card/icons/index.js

```diff
@@ -2,9 +2,10 @@ import React from 'react'
 
 import BankAxept from './BankAxept'
 import DNB from './DNB'
-import Expired from './Clock'
-import Blocked from './Padlock'
-import NotActive from './CardIn'
+import { clock_medium as Expired } from '../../../icons'
+import { padlock_medium as Blocked } from '../../../icons'
+import { card_in_medium as CardIn } from '../../../icons'
+import { hourglass as Hourglass } from '../../../icons'
```

Comment:
```suggestion
import { clock_medium as Expired,  padlock_medium as Blocked, card_in_medium as CardIn hourglass as Hourglass } from '../../../icons'
```

Its possible to get all icons in one import.